ibm/service/database/resource_ibm_database.go

```diff
@@ -2837,6 +2870,39 @@ func validateGroupHostFlavor(groupId string, resourceName string, group *Group)
 	return nil
 }
 
+func validateMultitenantMemoryCpu(groupId string, resourceName string, resourceDefaults *Group, group *Group) error {
```

Comment:
Do you need `resourceName` parameter here?

Reply:
Nope! Thank you for catching that 😄